code/go/0chain.net/blobbercore/filestore/fs_store.go

```
@@ -56,7 +56,9 @@ type StoreAllocation struct {
 }
 
 func SetupFSStore(rootDir string) FileStore {
-	createDirs(rootDir)
+	if err := createDirs(rootDir); err != nil {
```

probably should fail early as well?